TESTING.md

```diff
@@ -64,7 +64,7 @@ This test suite is based on [BackstopJS](https://garris.github.io/BackstopJS) pl
 Documentation available [here](https://github.com/garris/BackstopJS#backstopjs)
 
 ### Requirements
-A CiviCRM with pre-configured with CiviCRM, Shoreditch, CiviCase, and any other dependencies. [See how](https://github.com/compucorp/uk.co.compucorp.civicase#installation-civibuild)
+A CiviCRM with pre-configured CiviCRM, Shoreditch, CiviCase, and any other dependencies. [See how](https://github.com/compucorp/uk.co.compucorp.civicase#installation-civibuild)
```

Comment:
I am sorry but this still does not sound right
`A CiviCRM with pre-configured CiviCRM, Shoreditch, CiviCase, and any other dependencies`

notice the CiviCRM is present twice.